renderdoc/driver/gl/wrappers/gl_buffer_funcs.cpp

```diff
@@ -1167,6 +1210,13 @@ void WrappedOpenGL::glBindBufferBase(GLenum target, GLuint index, GLuint buffer)
 {
   ContextData &cd = GetCtxData();
 
+  if(IsBackgroundCapturing(m_State) && target == eGL_TRANSFORM_FEEDBACK_BUFFER &&
+     RecordUpdateCheck(cd.m_FeedbackRecord))
```

Comment:
I don't necessarily mind if you want to move the frame references above the GL call for all resource types - including the transform feedback object here - but if you want to do that you should do it consistently. E.g. it hasn't been done for sampler objects.

To reduce the number of changes etc I'm fine with only doing this for buffers and textures, and not other resources like this that we don't plan on postponing.

Reply:
Yeah, good point. I think it really makes sense to move incrementally, and don't try to cover everything in one pass. This will make reviews easier, and would be less error-prone. I agree, we should reduce it only to buffers and texture, will work on it today.